superset/charts/schemas.py

```diff
@@ -854,6 +854,7 @@ class ChartDataExtrasSchema(Schema):
     having_druid = fields.List(
         fields.Nested(ChartDataFilterSchema),
         description="HAVING filters to be added to legacy Druid datasource queries.",
+        deprecated=True,
```

Comment:
I think we can go ahead and remove these.. the integration is already broken when we removed the DRUID config.